server.js

```diff
@@ -0,0 +1,102 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((request, response) => {
+  const uri = request.url;
+  const reqMethod = request.method;
+  const reqVersion = request.httpVersion;
+
+  function readF(fileLocation, conType, statusCode, reasonPhrase) {
+    fs.readFile(fileLocation, (err, data) => {
+      if (err) throw err;
```

Comment:
Your need to actually use your Error handling. If you try to read a non-existing file an error is throw. You need to use that error to know the file does not exist and trigger the steps to return a 404.